### ISSUE TITLE
59437X26 - Slender: The Arrival Demo

### ISSUE THREAD

**Issue by heysehaven** [xenia-project/game-compatibility#197]
[Marketplace] (http://marketplace.xbox.com/Product/Slender-The-Arrival/66acd000-77fe-1000-9115-d80258411430?downloadtype=GameDemo&purchase=1?cid=majornelson)
Test with EmuCr Xenia Git (2015/09/22)
Hangs and then it crashes 
[Crash Log] (https://gist.github.com/heysehaven/c01c9072c0414293d442)

**Comment by Razzile** [comment]
are you sure that is the correct ID for the game?

**Comment by heysehaven** [comment]
@Razzile should I change to "Trial" ?
![trial](https://cloud.githubusercontent.com/assets/14338853/10119306/f453b9cc-6468-11e5-9063-86cfd2cab847.PNG)

**Comment by Razzile** [comment]
no I mean the 59437X26. I think they shouldn't have letters above F in

**Comment by DrChat** [comment]
@Razzile @heysehaven Yeah, IDs are hexadecimal. Should be in the range 0-9 A-F.
I think all of heysehaven's IDs might be wrong on the issues list.

**Comment by SonofUgly** [comment]
It looks like the ID is in the 'Share this' link on the marketplace, so I'm guessing this should be 81BFE91F.

**Comment by DrChat** [comment]
That's wrong too. Should be the last 8 digits of the marketplace link, which should match what's logged by Xenia.

**Comment by SonofUgly** [comment]
The share link is a link to the marketplace. heysehaven posted the the full version's link and not the trial version. Also the trial/demo links are redirected to the main game's marketplace URL.
http://marketplace.xbox.com/en-US/Product/Trial-Game-Slender-The-Arrival/8a51d8b5-9f61-4bfb-b658-bd5d81bfe91f?cid=SLink

**Comment by gibbed** [comment]
This is an XBLA game so the full game and demo are the same download and share title IDs (you can confirm this by the fact that they have the same file size listed on the marketplace).

**Comment by ZolaKluke** [comment]
Tested on https://github.com/xenia-project/xenia/commit/57e8f05234f2cf8ee29cffd3ab8fe57de38479b8

Log: 
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2855158/xenia.zip)

Screenshots:
![image](https://user-images.githubusercontent.com/6497775/52627074-b71ebe80-2f08-11e9-97ac-66df69716f52.png)

Labels:
state-crash-host

**Comment by ZolaKluke** [comment]
Tested on: https://github.com/ZolaKluke/xenia/commit/4e78d5697f1a870b407b79947f1436222669218e
Official equivalent: https://github.com/xenia-project/xenia/commit/5e30f3a947426363f10ce24ea2954b99aa26b676

Nothing has changed at all in the past 2 months
![image](https://user-images.githubusercontent.com/6497775/55886904-efccd280-5bef-11e9-8cbd-33d13d86be8b.png)

Log: 
[xenia.log](https://github.com/xenia-project/game-compatibility/files/3064266/xenia.log)

Labels:
state-crash-host

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e

Played for 2 minutes and 41 seconds before noping out. 80-110FPS with vsync off and shaders cached.

:warning:You must set `break_on_unimplemented_instructions=false` and `protect_zero=false`!

![sle (5)](https://user-images.githubusercontent.com/31154606/105615520-fd3c5800-5d96-11eb-88ed-7f7ed7ff3f66.png)

<details>
  <summary>More Screenshots</summary>

![sle (1)](https://user-images.githubusercontent.com/31154606/105615523-02010c00-5d97-11eb-82ba-56765d4dc779.png)
![sle (2)](https://user-images.githubusercontent.com/31154606/105615524-02010c00-5d97-11eb-88ef-d68d3c2b5cea.png)
![sle (3)](https://user-images.githubusercontent.com/31154606/105615525-0299a280-5d97-11eb-89bc-5e7e90dc86b7.png)
![sle (4)](https://user-images.githubusercontent.com/31154606/105615527-0299a280-5d97-11eb-99ae-f9509917a3f6.png)
![sle (6)](https://user-images.githubusercontent.com/31154606/105615528-03323900-5d97-11eb-8627-7ad60af4df65.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5861201/xenia.zip)

state-gameplay

**Comment by Pepper134** [comment]
tested on  [xenia canary](url)

Played through and beat the prologue without issues.  30 FPS with Vsync on and shaders cached. You have to set break_on_unimplemented_instructions=false and protect_zero=false Full game should work properly now.
